debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -1053,7 +1054,8 @@ protected Duration xminFetchInterval() {
                     SSL_SOCKET_FACTORY,
                     STATUS_UPDATE_INTERVAL_MS,
                     TCP_KEEPALIVE,
-                    XMIN_FETCH_INTERVAL)
+                    XMIN_FETCH_INTERVAL,
+                    CommonConnectorConfig.SKIPPED_OPERATIONS)
```

Comment:
The config property is already enabled in the CommonConnectorConfig.CONFIG_DEFINITION which leads to the build error.

@gunnarmorling @jpechane should we disable/remove CommonConnectorConfig.SKIPPED_OPERATIONS from CommonConnectorConfig.CONFIG_DEFINITION and enable it only for the connectors where it's implementation is already finished? It seems like it's not part of the debezium-core ChangeRecordEmitter, which would maybe make sense.

Reply:
Yeah, I got this error too. I didn't get it why it should be happening. I blame myself for poor knowledge of java :)))

Reply:
I didn't get this error. wierd

Reply:
```[ERROR] shouldValidateConnectorConfigDef(io.debezium.connector.postgresql.PostgresConnectorIT)  Time elapsed: 0.013 s  <<< ERROR!

org.apache.kafka.common.config.ConfigException: Configuration skipped.operations is defined twice.
	at io.debezium.connector.postgresql.PostgresConnectorIT.shouldValidateConnectorConfigDef(PostgresConnectorIT.java:128)
```

You should get this error, maybe you just didn't realized it?
It's not specific to Java but more related to like we manage the CONFIG_DEFINITION since some time. So don't worry!
Still, I want to double-check with my team how we resolve it.

Reply:
I created https://issues.redhat.com/browse/DBZ-2699 to follow up on it.

Reply:
I started watching the issue. thanks

Reply:
Thx, can you remove that unnecessary line `CommonConnectorConfig.SKIPPED_OPERATIONS` then?
We will handle the rest in that task.

Reply:
Yeah! I will. if I can help with this issue please ping me